### PR TITLE
Update Hyperterm to Hyper

### PR DESCRIPTION
**Pull request by thibmaek**
See https://github.com/zeit/hyper/releases/tag/0.8.1

### CODE REVIEW

File: Casks/hyperterm.rb [hunk 1]
```diff
@@ -1,26 +1,26 @@
 cask 'hyperterm' do
```
Comment (vitorgalvao) [inline Casks/hyperterm.rb]: This also needs to change (and the name of the file).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

File: Casks/hyper.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+cask 'hyper' do
+  version '0.8.1'
+  sha256 '9b39e2dc710845db99ad2002e107c0c7754b7718c6cc41c8793435e9d18c3e03'
+
+  # github.com/zeit/hyper was verified as official when first introduced to the cask
+  url "https://github.com/zeit/hyper/releases/download/#{version}/hyper-#{version}-mac.zip"
+  appcast 'https://github.com/zeit/hyper/releases.atom',
+          checkpoint: '5ccfd2a1a40f2a2c2ea39a554f122fc165a2815de4c2d864c11c2153981088e6'
```
Comment (stovmascript) [inline Casks/hyper.rb]: Seems like the releases.atom file sha256 checksum also needs to be updated:

```shell
$ wget https://github.com/zeit/hyper/releases.atom
$ shasum -a 256 releases.atom 
9cb83c950deb477b1d9c95ebecbdaba3637db29007b2ff279ec4301147426b49  releases.atom
```

Comment (vitorgalvao) [inline Casks/hyper.rb]: @stovmascript Even though in this case it’s the same, that not how you calculate [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 9e5b3db2d122990b81e9686903577379e769f963. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**